src/page-modules/contact/components/input/searchable-select.tsx

```diff
@@ -0,0 +1,147 @@
+        inputValue.toLowerCase().replace(/\s+/g, ''), // Remove all white spaces from input value
+      ),
+  );
+
+  const getOptionBySelector = (key: string) =>
+    options.find((option) => option.id.toLowerCase() === key.toLowerCase());
+
+  const getOptionBySearch = (input: string) => {
+    return options.find(
+      (option) => option.name.toLowerCase() === input.toLowerCase(),
+    );
+  };
+
+  const handleSelectionChange = (key: Key | null) => {
+    if (key === null) return;
+    const keyString = key.toString();
+    const option = getOptionBySelector(keyString);
+    setInputValue(option?.name || keyString);
+    onChange(option?.value);
+  };
+
+  const handleInputChange = (inputValue: string) => {
+    const option = getOptionBySearch(inputValue);
+    setInputValue(option?.name || inputValue);
+    onChange(option?.value);
+  };
+
+  return (
+    <ComboBox
+      items={filteredOptions as Iterable<object>}
+      isDisabled={isDisabled}
+      onSelectionChange={handleSelectionChange}
+      className={style.searchable_select__comboBox}
+    >
+      <Label>{label}</Label>
+      <Group
+        className={andIf({
+          [style.searchable_select__group]: true,
+          [style.searchable_select__disabled]: !!isDisabled,
+        })}
+      >
+        <Input
+          placeholder={placeholder}
+          className={style.searchable_select__input}
+          value={inputValue}
+          onChange={(e) => handleInputChange(e.target.value)}
+        />
+        <Button className={style.searchable_select__button}>
+          <MonoIcon icon={'navigation/ExpandMore'} />
+        </Button>
+      </Group>
+      <Popover className={style.searchable_select__popover}>
+        <ListBox className={style.searchable_select__listBox}>
+          <ListBoxItem
+            textValue={placeholder}
+            isDisabled
+            className={style.searchable_select__listBoxItem}
+          >
+            <div className={style.searchable_select__item}>
+              <span className={style.searchable_select__checkmark}>
+                {!value ? '✔' : '\u00A0'}
+              </span>
+              <span className={style.searchable_select__truncate}>
+                {placeholder}
+              </span>
+            </div>
+          </ListBoxItem>
+          {filteredOptions.map((item) => (
+            <ListBoxItem
+              id={item.id}
+              key={item.id}
+              textValue={item.name}
+              className={style.searchable_select__listBoxItem}
+            >
+              {({ isSelected, isFocused }) => (
+                <div
+                  className={`${style.searchable_select__item} ${
+                    isFocused ? style.searchable_select__highlight : ''
+                  } ${isSelected ? style.searchable_select__selected : ''}`}
+                >
+                  <span className={style.searchable_select__checkmark}>
+                    {value && isSelected ? '✔' : '\u00A0'}
```

Comment:
Could we use the `Confirm.svg` instead here? https://github.com/AtB-AS/design-system/blob/main/packages/assets/files/common/mono/actions/Confirm.svg

Reply:
Sure, we can use `Confirm.svg` instead. Will update.